packages/go/src/go-helpers.ts

```diff
@@ -22,6 +22,7 @@ import type { Env } from '@vercel/build-utils';
 const streamPipeline = promisify(pipeline);
 
 const versionMap = new Map([
+  ['1.21', '1.21.0'],
```

Comment:
```suggestion
  ['1.21', '1.21.1'],
```

Let's use the latest version https://go.dev/dl/

Reply:
Hmm I thought I committed this but I guess I just "resolved it". fixing now